src/Containers/Lobby/index.tsx

```diff
@@ -15,16 +16,27 @@ const Lobby = () => {
   const { resources } = useContext(Resources);
   const { players, owner } = resources;
 
+  const writeToClipboard = async () => {
+    try {
+      await navigator.clipboard.writeText(gameCode);
+      pushNotification('info', 'Game code copied to clipboard');
+    } catch {
+      pushNotification('error', 'Game code cannot be copied');
+    }
+  }
+
   return (
     <Box>
       <CenteredContainer>
         <Row gutter={[0, 15]}>
           <Col span={16} offset={4}>
             <Row gutter={[10, 20]}>
               <Col span={12}>
-                <Button onClick={() => navigator.clipboard.writeText(gameCode)} style={{ fontFamily: 'Arial' }}>
-                  {gameCode}
-                </Button>
+                <Tooltip title='Click to copy game code'>
+                  <Button onClick={writeToClipboard} style={{ fontFamily: 'Arial' }}>
```

Comment:
Although It doesn't matter in this specific case- please use arrow functions in callbacks (preventing improper content of `this`)
```suggestion
                  <Button onClick={() => writeToClipboard} style={{ fontFamily: 'Arial' }}>
```